assets/Meow/DynamicAudio.gd

```diff
@@ -0,0 +1,81 @@
+extends Node
+
+var debug = true
+var musicOn = true
+var masterVol = -10
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	var dirMusic = "res://Meow/Music/Reg/"
+	#dir_contents(dirMusic)
+	var player = AudioStreamPlayer.new()
+	self.add_child(player)
+	player.volume_db = masterVol
+	while (musicOn):
+		player.stream = load(dirMusic+"/"+get_audio(dirMusic,random(dir_counter(dirMusic))))
```

Comment:
it doesn't really matter that much for the purpose of the project, but I think this is a good opportunity to try making this code more efficient - right now, we have to scan all of the files in the music directory (in dir_counter) each time a new song is picked; is there a way we can scan the folders just once? 

extra credit 1: Is there a way we can avoid having to scan through the folders again when we call get_audio?

I don't want to make this a chore, so if you don't feel like refactoring the code it's nbd